test/predicates/MintableERC721Predicate.test.js

```diff
@@ -9,6 +9,7 @@ import * as deployer from '../helpers/deployer'
 import { mockValues } from '../helpers/constants'
 import logDecoder from '../helpers/log-decoder.js'
 import { getERC721TransferLog, getERC721TransferWithMetadataLog } from '../helpers/logs'
+import { it } from 'ethers/wordlists'
```

Comment:
this is going to break tests, remove it please

Reply:
Yeah, done